input/resources/au-core-requester.xml

```diff
@@ -854,6 +854,7 @@
 <blockquote>
 <div>
 <p>If the requester supports the Encounter resource, the requester <strong>SHALL</strong> support the AU Core profile and the conformance expectations for the Encounter resource.</p>
+<p>The Encounter resource can communicate the clinical indication using either a code in Encounter.reasonCode or a reference using Encounter.reasonReference. Although both <code>Encounter.reasonCode</code> and <code>Encounter.reasonReference</code> are marked as Must Support, a responder is not required to support both but <strong>SHALL</strong> support at least one of these elements. The requester <strong>SHALL</strong> support both elements.</p>
```

Comment:
Apply formatting to Encounter.reasonCode and Encounter.reasonReference in the first sentence as well.

Reply:
fixed in all instances.